### PR TITLE
Track strimzi kafka/kafka connect status in ClowdEnvironment deployent status

### PR DESCRIPTION
**Pull request by kylape**
Had to revert this change because the Kafka resource in stage (and prod) is in an old format that we need to update.  We can merge this again once the format has been updated.  Specifically kafka.spec.kafka.listeners.  I have a quick test that I should work through to make sure there aren't any other fields to fix.

### CODE REVIEW

File: controllers/cloud.redhat.com/utils/utils.go [hunk 1]
```diff
@@ -234,44 +234,22 @@ func MakePVC(pvc *core.PersistentVolumeClaim, nn types.NamespacedName, labels ma
...
-			// Since there's no replicas, there's nothing to do to get ready
-			return true
-		}
-
-		if deployment.Status.UpdatedReplicas < *deployment.Spec.Replicas {
-			// This indicates that the deployment has not completed rolling out the new ReplicaSet
-			return false
-		}
-	}
+// IntPtr returns a pointer to the passed integer.
```
Comment (psav) [inline controllers/cloud.redhat.com/utils/utils.go:237]: These have been moved - can you use the ones in `common.go`?

**Review by psav — CHANGES_REQUESTED** [review]

File: controllers/cloud.redhat.com/utils/utils.go (hunk 1 above)
Comment (bsquizz) [reply controllers/cloud.redhat.com/utils/utils.go:237]: This particular func wasn't moved. Want me to move it as part of this PR?

```
 grep IntPtr -r *
controllers/cloud.redhat.com/providers/kafka/appinterface.go:			Port:     utils.IntPtr(9092),
controllers/cloud.redhat.com/providers/kafka/localkafka.go:			Port:     utils.IntPtr(29092),
controllers/cloud.redhat.com/providers/web/default.go:	c.WebPort = utils.IntPtr(int(web.Env.Spec.Providers.Web.Port))
controllers/cloud.redhat.com/providers/web/default.go:	c.PublicPort = utils.IntPtr(int(web.Env.Spec.Providers.Web.Port))
controllers/cloud.redhat.com/providers/web/default.go:	c.PrivatePort = utils.IntPtr(int(privatePort))
controllers/cloud.redhat.com/utils/utils.go:// IntPtr returns a pointer to the passed integer.
controllers/cloud.redhat.com/utils/utils.go:func IntPtr(i int) *int {
```

Comment (kylape) [reply controllers/cloud.redhat.com/utils/utils.go:237]: Can this be a follow up item?

**Comment by kylape** [comment]
lol I can't approve this because technically I submitted it

**Comment by psav** [comment]
This was merged in another PR

**Pull request closed without merging**